ci/gpu/build.sh

```diff
@@ -117,18 +121,21 @@ else
         TEST_MODE_FLAG=""
     fi
 
+    gpuci_logger "Running cuGraph test.sh..."
     ${WORKSPACE}/ci/test.sh ${TEST_MODE_FLAG} | tee testoutput.txt
+    CMDRETCODE=$?; EXITCODE=$((EXITCODE | $CMDRETCODE))
+    gpuci_logger "Ran cuGraph test.sh : return code was: $CMDRETCODE, gpu/build.sh exit code is now: $EXITCODE"
 
-    echo -e "\nTOP 20 SLOWEST TESTS:\n"
-    # Wrap in echo to prevent non-zero exit since this command is non-essential
-    echo "$(${WORKSPACE}/ci/getGTestTimes.sh testoutput.txt | head -20)"
-
+    gpuci_logger "Running cuGraph notebook test script..."
     ${WORKSPACE}/ci/gpu/test-notebooks.sh 2>&1 | tee nbtest.log
+    CMDRETCODE=$?; EXITCODE=$((EXITCODE | $CMDRETCODE))
+    gpuci_logger "Ran cuGraph notebook test script : return code was: $CMDRETCODE, gpu/build.sh exit code is now: $EXITCODE"
     python ${WORKSPACE}/ci/utils/nbtestlog2junitxml.py nbtest.log
 fi
 
 if [ -n "\${CODECOV_TOKEN}" ]; then
```

Comment:
```suggestion
if [ -n "${CODECOV_TOKEN}" ]; then
```

This isn't related to your changes, but can you remove this erroneous `\`?

Reply:
fixed in next commit